### DISCUSSION TITLE
[Bug]Ran into compiling errors while trying to build WebRTCiOSSDK

### DISCUSSION THREAD

**Discussion by jim9586** [ant-media/Ant-Media-Server#5559] [Q&A]
Ran into the below compiling error while trying to build the project. Did nothing but just imported WebRTCiOSSDK
<img width="252" alt="Screenshot 2023-09-15 at 7 17 31 PM" src="https://github.com/ant-media/Ant-Media-Server/assets/5533663/a18169c6-2b17-4af5-87b5-69eef6a666bc">

Swift and XCode version: swift-driver version: 1.75.2 Apple Swift version 5.8.1 (swiftlang-5.8.0.124.5 clang-1403.0.22.11.100)
Target: arm64-apple-macosx13.0

**Comment by tremzo** [comment]
@joeAI335 I reproduced the error by following the tutorial here https://antmedia.io/build-and-try-webrtc-ios-app-for-dummies/ and trying to run it.

<img width="1577" alt="Screenshot 2023-09-18 at 10 06 32" src="https://github.com/ant-media/Ant-Media-Server/assets/121940452/59ced54e-6919-4c5a-ad7e-b3897efd2741">


Let's create an issue for this.

**Comment by tremzo** [comment]
Here is the issue @joeAI335 https://github.com/ant-media/Ant-Media-Server/issues/5565

**Comment by mekya** [comment]
Hi @joeAI335 @timantmedia 

The fix has been merged to the master

Please try and let me know if it works for you

Cheers
Oguz
